core/src/main/resources/hudson/model/Api/index.jelly

```diff
@@ -98,58 +98,47 @@ THE SOFTWARE.
         -->
       </dl>
 
-      <p>
-        For more information about remote API in Jenkins, see
-        <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.
-      </p>
+      <p>For more information about remote API in Jenkins, see <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.</p>
```

Comment:
@recena Integrating obviously unrelated changes into a single PR slows down review and merge of the real fix. It is convenient neither for reviews nor for the committer himself. Any reviewer may **legitimately** block the PR due to unrelated changes, so by adding them you accept the risk of such blockage. Just IMHO.